File: BEE 1101.py
while(True):
    m, n = map(int, input().split(" "))
    if (m == 0 or m < 0 or n == 0 or n < 0):
            break
    
    if (m < n):
        temp = m
        m = n
        n = temp

    sum = 0
    for i in range(n, m+1):
        sum += i
        print(i, end=" ")
    
    print(f"Sum={sum}")

File: BEE 1101.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

while (true){
    var m = parseInt(line.shift());
    var n = parseInt(line.shift());
    if (m === 0 || m < 0 || n === 0 || n < 0){
        break;
    }

    if (m < n){
        var temp = m;
        m = n;
        n = temp;
    }

    var sum = 0;

    for (var i = n; i <= m ; i ++){
        sum += i;
        process.stdout.write(i + " ");
    }
    console.log("Sum="+sum);
}